### PR TITLE
Add comments search api method

### PR DESCRIPTION
**Pull request by bujhm131313**
Implement comments search method

### CODE REVIEW

**Comment by anxolerd** [comment]
I believe, this can be a part of objects API. @shchotse, can you please take a look?

**Comment by shchotse** [comment]
agree with @anxolerd, in case with comments for object entity, they can be a part of the object service.
but if you want to make isolated service for all comments and all entities, then you should additionally rename the `search` method, for maybe `search_by_object_id`.

**Comment by bujhm131313** [comment]
api point `/objects/{object_id}/comments.json` looks like part of object service for searching for tickets by connected object_id, so I guess I will rename it to `search_by_object_id`

File: useresponse/api/comments.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class CommentService(object):
+    """Service which contains comment-related API calls"""
+
+    def __init__(self, transport):
+        self._transport = transport
+
+    def search_by_object_id(
+        self,
+        object_id: int = 0,
+        is_private: Optional[int] = 0,
```
Comment (anxolerd) [inline useresponse/api/comments.py]: Please, make it boolean at this level of abstraction and transform to 0 or 1 inside the function.

**Review by anxolerd — CHANGES_REQUESTED** [review]

File: useresponse/api/comments.py (hunk 1 above)
Comment (bujhm131313) [reply useresponse/api/comments.py]: Done, tnx

**Review by anxolerd — APPROVED** [review]

**Pull request merged**